lib/spack/spack/binary_distribution.py

```diff
@@ -127,7 +128,18 @@ def write_buildinfo_file(prefix, workdir, rel=False):
             #  Check if the file contains a string with the installroot.
             #  This cuts down on the number of files added to the list
             #  of files potentially needing relocation
```

Comment:
The comment above needs an update -- you could work it into the branches of the if for more clarity